src/GroupAction/GroupActionCollection.php

```diff
@@ -155,12 +191,54 @@ public function submitted(NetteForm $form): void
 		$ids = array_keys($http_ids);
 
 		$id = $values->group_action;
-		$this->groupActions[$id]->onSelect($ids, $values[$id] ?? null);
+
+		if ($submitter->getName() === 'submit') {
+			$this->groupActions[$id]->onSelect($ids, $values[$id] ?? null);
+		} else {
+			$submitter->onClick($ids);
+		}
 
 		$form['group_action']['group_action']->setValue(null);
 	}
 
 
+	private function getFormSubmitter(NetteForm $form): ?Nette\Forms\Controls\SubmitButton
+	{
+		if (
+			isset($form['group_action']['submit'])
+			&& $form['group_action']['submit']->isSubmittedBy()
+		) {
+			$form['group_action']['submit'];
+		}
+
+		foreach ($form['group_action']->getComponents() as $component) {
+			if (
+				$component instanceof Nette\Forms\Controls\SubmitButton
+				&& $component->isSubmittedBy()
+			) {
+				return $component;
+			}
+		}
+
+		return null;
+	}
+
+
+	/**
+	 * Add one group button action to collection of actions
+	 */
+	public function addGroupButtonAction(string $title, ?string $class = null): GroupAction
+	{
+		if (count($this->groupActions) > 0) {
```

Comment:
Could you please use `if ($this->groupActions !== []) {`? It is a bit faster then count, thanks

Reply:
It's for consistence with code around this. Potential for refactoring..? :-)